<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClient</class>
 <widget class="QMainWindow" name="ChatClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>401</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_input">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser_output">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_send">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_closeConn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reconnect">
         <property name="text">
          <string>ReConnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Server"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Server">
   <property name="text">
    <string>&amp;Server</string>
   </property>
   <property name="toolTip">
    <string>Server Connect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Server</sender>
   <signal>triggered()</signal>
   <receiver>ChatClient</receiver>
   <slot>inputServerDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reconnect</sender>
   <signal>clicked()</signal>
   <receiver>ChatClient</receiver>
   <slot>connectToServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_send</sender>
   <signal>clicked()</signal>
   <receiver>ChatClient</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_input</sender>
   <signal>returnPressed()</signal>
   <receiver>ChatClient</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputServerDetails()</slot>
  <slot>connectToServer()</slot>
  <slot>sendClicked()</slot>
  <slot>stopClicked()</slot>
 </slots>
</ui>
